cypress/e2e/tests/footer.cy.js

```diff
@@ -0,0 +1,220 @@
+context('Footer Get Started', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('div')
+      .contains('Get Started')
+      .siblings('div')
+      .as('getStarted')
+  })
+
+describe('Voice', () => {
+  it('Should verify the external link exists', () => {
+    cy.get('@getStarted')
```

Comment:
+1 on using existing markup to do additional checks.

Reply:
Thanks!